server/views/eligibility-checker/cannot-trade.html

```diff
@@ -16,6 +16,17 @@ <h1 id="pageTitle" class="govuk-heading-l">{{ pageTitle }}</h1>
 
       <h2 id="heading2" class="govuk-heading-m">What you can do with this item</h2>
 
+
+      <p id="helpText3p">Based on your answers, it is illegal to:</p>
```

Comment:
Add unit test for this element (check existence and value). Also I'm not sure about the ID here, would ".....2b' be better (i.e. why does it end in a 'p'?)